index.html

```diff
@@ -1,20 +1,52 @@
-﻿<!DOCTYPE HTML>
+<!DOCTYPE HTML>
 <html lang="pl">
 <head>
 
 	<meta charset="utf-8" />
-	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-	
-	<title></title>
-	
-	<meta name="description" content="" />
-	<meta name="keywords" content="" />
-	
+	<title>
+        Hangman - Wisielec
+    </title>
+
+    <link 
```

Comment:
`link` and `scripts` can be attached inline instead of dividing it per attribute:
`<link rel="..." href="..." type="...">`